main/http_server/axe-os/src/app/components/home/home.component.ts

```diff
@@ -360,4 +367,22 @@ export class HomeComponent implements OnInit, OnDestroy {
 
     return this.calculateAverage(efficiencies);
   }
+
+  public handleSystemMessages(info: ISystemInfo) {
+    const updateMessage = (condition: boolean, severity: Message['severity'], text: string) => {
```

Comment:
I'm confused by the `severity: Message['severity']`. What does the Message struct reference doing here?

Isn't this just a function with 3 arguments?